server.js

```diff
@@ -32,5 +33,7 @@ knex1.migrate.latest().then(() => {
     log.info(`Server listening on port ${app.get('port')}`)
   })
   // must be created after migration scripts are complete
-  sqsService.app.start()
+  if (config.sqs.enabled === 'true') {
+    sqsService.app.start()
+  }
```

Comment:
defaults to true - can now use SQS_ENABLED = false in env to disable sqs consumer and avoid the log errors

Reply:
👍